src/constants.cpp

```diff
@@ -1904,7 +1904,15 @@ struct spell_types spells[] =
     { "Ice Sheet", TRUE, MANIPULATION, SINGLE, -1, INSTANT, 1, SERIOUS },
     { "Ignite", TRUE, MANIPULATION, SINGLE, -1, PERMANENT, 1, DEADLY },
     { "Light", TRUE, MANIPULATION, SINGLE, -1, SUSTAINED, 2, MODERATE },
-    { "Shadow", TRUE, MANIPULATION, SINGLE, -1, SUSTAINED, 2, MODERATE }
+    { "Shadow", TRUE, MANIPULATION, SINGLE, -1, SUSTAINED, 2, MODERATE },
+    { "Laser", TRUE, MANIPULATION, SINGLE, -1, INSTANT, 1, -2 },
+    { "Nova", TRUE, MANIPULATION, AREA, -1, INSTANT, 1, -1 },
+    { "Steam", TRUE, MANIPULATION, SINGLE, -1, INSTANT, 0, -2 },
+	{ "Smoke Cloud", TRUE, MANIPULATION, AREA, -1, INSTANT, 1, -1 },
+    { "Thunderbolt", TRUE, MANIPULATION, SINGLE, -1, INSTANT, 0, -2 },
+	{ "Thunderclap", TRUE, MANIPULATION, AREA, -1, INSTANT, 1, -1 },
+	{ "Waterbolt", TRUE, MANIPULATION, SINGLE, -1, INSTANT, 0, -3 },
+	{ "Splash", TRUE, MANIPULATION, AREA, -1, INSTANT, 0, -3 },
```

Comment:
A comma at the end indicates there's another line to come after it, so since this is the last spell that should be removed.

Reply:
Comma removed